### ISSUE TITLE
WordPress usage

### ISSUE THREAD

**Issue by Opussci** [rstudio/distill#127]
Hi,
I'm wondering is it possible to add distill framework/templating to the WordPress sites?
Thanks

**Comment by jjallaire** [comment]
No, that's an entirely different effort that wouldn't be well integrated w/
the tooling of either core Distill or Distill for R Markdown.

**Comment by Opussci** [comment]
My workflow would be just to serve static html from inside WordPress. After Authoring it in R markdown

**Comment by jjallaire** [comment]
There are a few articles/tutorials on the web that address this, e.g.
http://tobiasdienlin.com/2019/03/08/how-to-publish-a-blog-post-on-wordpress-using-rmarkdown/

**Comment by Opussci** [comment]
Thanks for link, I read it before. Still, references can not be parsed in that way.
